extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -9,203 +12,306 @@ class SpecialDiscussionsLogController extends WikiaSpecialPageController {
 	//2. Add date range input field
 
 	const PAGINATION_SIZE = 50;
+	const REQUEST_SIZE = 200;
 	const DAYS_RANGE = 15;
 	const HTTP_STATUS_OK = 200;
-	const NO_USER_MATCH_ERROR = "Provided username did not match any user";
+	const NO_USER_MATCH_ERROR = 'discussionslog-no-user-match-error';
 
 	const DEFAULT_TEMPLATE_ENGINE = \WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
 
+	private $logger;
+	private $users;
+
 	public function __construct() {
 		parent::__construct( 'DiscussionsLog', '', false );
+		$this->users = [ ];
+		$this->logger = Wikia\Logger\WikiaLogger::instance();
 	}
 
 	public function index() {
 
-		if ( !$this->checkAccess() ) {
-			throw new \PermissionsError( 'specialdiscussionslog' );
-		}
+		$this->checkAccess();
 
 		$this->setHeaders();
+		$this->response->addAsset( '/extensions/wikia/SpecialDiscussionsLog/css/DiscussionsLog_Forms.scss' );
```

Comment:
This works, but I think the preferred way to do this is to create a new entry in [AssetsManager/config.php](https://github.com/Wikia/app/blob/dev/extensions/wikia/AssetsManager/config.php) and then add the asset via the name in the config. Like [this](https://github.com/Wikia/app/blob/dev/extensions/wikia/UserActivity/SpecialUserActivityController.class.php#L44).